webapp/src/containers/Rhs/projectDetails/index.tsx

```diff
@@ -137,14 +147,16 @@ const ProjectDetails = (projectDetails: ProjectDetails) => {
     const {isLoading: isUnlinkProjectLoading} = getApiState(plugin_constants.pluginApiServiceConfigs.unlinkProject.apiServiceName, projectDetails);
     const {isLoading: isDeleteSubscriptionLoading} = getApiState(plugin_constants.pluginApiServiceConfigs.deleteSubscription.apiServiceName, subscriptionToBeDeleted);
 
+    // Update the subscription list on RHS when a subscription is deleted using the slash command
+    useEffect(() => {
+        if (getWebsocketEventState(state).isSubscriptionDeleted) {
+            fetchSubscriptionList();
+            dispatch(toggleIsSubscriptionDeleted(false));
+        }
+    }, [getWebsocketEventState(state).isSubscriptionDeleted]);
+
     return (
         <>
-            <BackButton onClick={handleResetProjectDetails}/>
-            <ToggleSwitch
-                active={showAllSubscriptions}
-                onChange={setShowAllSubscriptions}
-                label={plugin_constants.common.ToggleLabel}
-            />
```

Comment:
this part is updated in master

Reply:
many changes are not in sync with the master. please check if this is correct

Reply:
okay will check when syncing with master

Reply:
this is left to be synced.

Reply:
this is fine for now in current branch, please update `getCurrentChannelSubscriptions` here